### ISSUE TITLE
NVDA + MathCAT reads math elements twice in a PDF produced from LaTeX using math/setup={mathml-AF, mathml-SE}

### ISSUE THREAD

**Issue by pseeburger** [NSoiffer/MathCAT#521]
When I compile my Overleaf latex document to a PDF/UA-2 using the following preamble with math/setup={mathml-AF,mathml-SE}, NVDA+MathCAT reads all math expressions two times when opened in FireFox.

`\DocumentMetadata{
  lang=en-US,
  pdfstandard=ua-2,
  pdfversion=2.0,
  testphase={phase-III,math},
  tagging-setup = {math/setup={mathml-AF,mathml-SE},
                   extra-modules={verbatim-mo},
                   table/header-rows=1}
}`

Is there a way to avoid this double reading, while keeping both versions of mathml in the PDF?

The mathml-SE is needed by Adobe Acrobat when NVDA+MathCAT are used to read the equations in the PDF.  It does not read the math correctly at all with only mathml-AF in place.

But in FireFox, the mathml-AF produces cleaner, clearer reading of the math, e.g., a piecewise function using the cases structure.

Is there a setting in MathCAT or in my latex compilation that would set a priority between the two versions and avoid this repetition?

If not, it seems that it may be helpful for MathCAT to automatically choose to read the mathml-AF version, if it were available and does not cause an error, and not read both versions.  It could then fallback to the mathml-SE version, if the AF version does not work.  There could be a setting to actually still read both versions (for testing purposes), but I can't imagine most people would find it helpful to hear the math read twice.
